client/data/transactions/resolvers.js

```diff
@@ -88,7 +88,7 @@ export function* getTransactions( query ) {
 	}
 }
 
-export function getTransactionsCSV( query ) {
+export function getTransactionsCSVRequestURL( query ) {
```

Comment:
Just a rename of prior work, to clarify that this returns a URL and not actual CSV.

Reply:
maybe just `downloadTransactions` or `exportTransactions` ? Because it may or may not return the URL ?

Reply:
That function we're talking about _only_ returns the URL/string. That's why I was confused by the previous name 😆

https://github.com/Automattic/woocommerce-payments/blob/18f5948e50badbccc91b6d2f48fafa04084706f4/client/data/transactions/resolvers.js#L91-L98

Reply:
Yes, but from the server it may or may not return the URL

Reply:
Ah, sorry I get what you mean now!